docs/intro.md

```diff
@@ -31,33 +30,70 @@ git clone https://github.com/substratusai/substratus
 cd substratus
 ```
 
-### Creating the infrastructure in GCP
+### Creating the infra and and and and deploying the controller
 
 Use our infrastructure build image to create a cluster and dependent cloud
 components:
 
 ```bash
-docker build ./infra -t substratus-infra && docker run -it \
-    -e REGION=us-central1 \
-    -e ZONE=us-central1-a \
-    -e PROJECT=$(gcloud config get project) \
-    -e TOKEN=$(gcloud auth print-access-token) \
-    substratus-infra gcp-up
+docker build ./install -t substratus-installer && \
+docker run -it \
+  -v $HOME/.kube:/root/.kube \
+  -e PROJECT=$(gcloud config get project) \
+  -e TOKEN=$(gcloud auth print-access-token) \
+  -e GPU_TYPE=nvidia-l4 \
```

Comment:
How is GPU_TYPE handled in the installer?

Reply:
It's not handled yet but will add that soon